__tests__/app.test.js

```diff
@@ -76,12 +76,55 @@ describe("GET /api/articles/:article_id", () => {
       expect(body.msg).toBe("Article Doesn't Exist");
     })
   })
-  test.only("400: sends an appropriate status and error message when given an invalid id", () => {
+  test("400: sends an appropriate status and error message when given an invalid id", () => {
     return request(app)
       .get('/api/articles/not-an-article-id')
       .expect(400)
       .then(({ body }) => {
         expect(body.msg).toBe('Bad Request');
       });
   });
-})
\ No newline at end of file
+})
+describe("GET /api/articles", () => {
+  test("200: Responds with all article objects with the body property not included and a comment_count property added", () => {
+     return request(app)
+     .get("/api/articles")
+     .expect(200)
+     .then(({ body }) => {
+      const { articles } = body;
+      ///console.log(articles)
+      expect(articles).toHaveLength(13)
+      articles.forEach((article) => {
+        expect(article).toMatchObject({
+          article_id: expect.any(Number),
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+          comment_count: expect.any(Number)
+        })
+      })
+    })
+  })
+ test("200: The articles should be sored by date in descending order", () => {
+  return request(app)
+  .get("/api/articles")
+  .expect(200)
+  .then(({ body }) => {
+    const { articles } = body;
+    expect(articles).toHaveLength(13);
+    const sortedArr = [...articles].sort((a, b) => b.created_at.localeCompare(a.created_at));
```

Comment:
Interesting approach, I like it! Is there any alternative methods we could have used here?